homeassistant/components/tibber/sensor.py

```diff
@@ -558,17 +558,22 @@ def __init__(self, hass, tibber_connection):
     async def _async_update_data(self):
         """Update data via API."""
         await self._tibber_connection.fetch_consumption_data_active_homes()
+        await self._tibber_connection.fetch_production_data_active_homes()
         await self._insert_statistics()
 
     async def _insert_statistics(self):
         """Insert Tibber statistics."""
         for home in self._tibber_connection.get_homes():
-            if not home.hourly_consumption_data:
-                continue
-            for sensor_type in (
-                "consumption",
-                "totalCost",
-            ):
+            sensors = []
+            if home.hourly_consumption_data:
+                sensors.append(("consumption", False, ENERGY_KILO_WATT_HOUR))
+                sensors.append(("totalCost", False, home.currency))
+            if home.hourly_production_data:
+                sensors.append(("production", True, ENERGY_KILO_WATT_HOUR))
+                sensors.append(("profit", True, home.currency))
+
+            for sensor in sensors:
```

Comment:
```suggestion
            for sensor_type, is_production, unit in sensors:
```

Reply:
No need for line 576 not 587, and small change to 648

Reply:
Fixed, thanks!